src/stacktrace_libunwind-inl.h

```diff
@@ -52,7 +52,7 @@ _START_GOOGLE_NAMESPACE_
 static bool g_now_entering = false;
 
 // If you change this function, also change GetStackFrames below.
-int GetStackTrace(void** result, int max_depth, int skip_count) {
+size_t GetStackTrace(void** result, size_t max_depth, int skip_count) {
   void *ip;
   int n = 0;
```

Comment:
Well, theoretically it is bad, but in this case it is totally harmless (no negative or very big numbers used here). If you want to be picky about that, there is also a few sign-compare warnings in symbolize.cc.